sucuri-cleanup.py

```diff
@@ -1,22 +1,115 @@
-# So far I'm mainly trying to collect the bits of code that I'm going to need for the core functionality of the script
-# this is by no means functional. And so far everything pretty much looks like a global. I'll work these bits down
-# into functions as I go, at least that's the plan so far ;)
+#ITS ALIVE!!!!!!!
+#And now, it's also more optimized. We've removed the nested for looping, and made some other minor enhancements.
+#The script now also checks to see if it's being run from an install's folder, and if not, it fails out with debug
+#Command line options for debug output and executing from a local command prompt have been added to streamline future additions to the code
+#did some cleanup to make the code prettier, and hopefully easier to read.
 
 import sys
 import os
 import requests
 import re
-
-salts = requests.get('https://api.wordpress.org/secret-key/1.1/salt/')
+import _mysql as dbcon
+debug = False
+local = False
+salts = requests.get('https://api.wordpress.org/secret-key/1.1/salt/').text
 pwd = os.getcwd()
 install = pwd.split('/')[4]
-
-def salt_replace()
+install_type = pwd.split('/')[3]
+def salt_replace():
     """
     Replace the salts in the installs wp-config.php file & grab the prefix while we're there.
     """
-    wp_config = open("/nas/content/live/"+ install + "/wp-config.php", r+)
-    regex_list=["AUTH_KEY","SECURE_AUTH_KEY","LOGGED_IN_KEY","NONCE_KEY","AUTH_SALT","SECURE_AUTH_SALT","LOGGED_IN_SALT","NONCE_SALT"]
-    regex=re.compile()
-    for line in wp_config:
+    salt_regex=re.compile("^define\('(AUTH_KEY|SECURE_AUTH_KEY|LOGGED_IN_KEY|NONCE_KEY|AUTH_SALT|SECURE_AUTH_SALT|LOGGED_IN_SALT|NONCE_SALT).*$")
+    regex_prefix=re.compile("\$table_prefix\s+=\s+\'(\w+_?)\';")
+    regex_dbuser=re.compile("define\(\s+\'DB_USER\',\s+\'(\w+)\'\s+\);")
+    regex_dbpass=re.compile("define\(\s+\'DB_PASSWORD\',\s+\'([A-Za-z0-9]+)\'\s+\);")
+    regex_dbname=re.compile("define\(\s+\'DB_NAME\',\s+\'(\w+)\'\s+\);")
+    output=""
+    os.rename("./wp-config.php", "./wp-config-security_backup.php")
+    with open("./wp-config-security_backup.php", "r+") as thing:
+        text = thing.read()
+    wp_config = open("./wp-config.php", "w")
+    text=text.rstrip().split('\n')
+    for index1,line in enumerate(text):
+        if re.search(salt_regex,line) != None:
```

Comment:
Just some constructive feedback here... but I generally try to shy away from double negatives. Sometimes you can't, but try to think of a way where you could rework this code to ask "If this IS something" rather than "If this is not nothing".

It really goes a long way for code readability -- maybe not a huge improvement for this context... but always keep that in the back of your mind. The next person who will be reading through this code will thank you! =D